dandi/cli/tests/test_download.py

```diff
@@ -18,6 +18,7 @@ def test_download_defaults(mocker):
         existing="error",
         format="pyout",
         jobs=6,
+        jobs_per_zarr=None,
```

Comment:
* I don't consider this parameter to be part of the public API, so we can just change the name if & when it starts to apply to more than just Zarrs.  Until then, YAGNI.
* This had to be added to the tests because they're testing the values passed to the mocked `upload()` function.

Reply:
ok, although it remains an ongoing discussion on what would/could be considered public API of a python package I guess. At least deprecation cycle for `jobs_per_zarr` -> `jobs_per_asset` migration would be relatively easy to setup.